From 3553cd1fd4ef2075edf3a2fdddbc8d7117effbb7 Mon Sep 17 00:00:00 2001
From: David Mercado <david.mercado@windriver.com>
Date: Tue, 19 Nov 2013 12:12:03 -0500
Subject: [PATCH 134/158] LSI AXM55XX SIM: Add "clock-frequency" to DTS

Add "clock-frequency" field to cpu entries to fix warnings at boot.

Signed-off-by: David Mercado <david.mercado@windriver.com>
---
 arch/arm/boot/dts/axm55xxsim.dts | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/arch/arm/boot/dts/axm55xxsim.dts b/arch/arm/boot/dts/axm55xxsim.dts
index f3dde14..1c6b413 100644
--- a/arch/arm/boot/dts/axm55xxsim.dts
+++ b/arch/arm/boot/dts/axm55xxsim.dts
@@ -45,96 +45,112 @@
 			device_type = "cpu";
 			compatible = "arm,cortex-a15";
 			reg = <0>;
+			clock-frequency= <0>;
 		};
 
 		cpu@1 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a15";
 			reg = <1>;
+			clock-frequency= <0>;
 		};
 
 		cpu@2 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a15";
 			reg = <2>;
+			clock-frequency= <0>;
 		};
 
 		cpu@3 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a15";
 			reg = <3>;
+			clock-frequency= <0>;
 		};
 
 		cpu@4 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a15";
 			reg = <4>;
+			clock-frequency= <0>;
 		};
 
 		cpu@5 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a15";
 			reg = <5>;
+			clock-frequency= <0>;
 		};
 
 		cpu@6 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a15";
 			reg = <6>;
+			clock-frequency= <0>;
 		};
 
 		cpu@7 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a15";
 			reg = <7>;
+			clock-frequency= <0>;
 		};
 
 		cpu@8 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a15";
 			reg = <8>;
+			clock-frequency= <0>;
 		};
 
 		cpu@9 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a15";
 			reg = <9>;
+			clock-frequency= <0>;
 		};
 
 		cpu@10 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a15";
 			reg = <10>;
+			clock-frequency= <0>;
 		};
 
 		cpu@11 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a15";
 			reg = <11>;
+			clock-frequency= <0>;
 		};
 
 		cpu@12 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a15";
 			reg = <12>;
+			clock-frequency= <0>;
 		};
 
 		cpu@13 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a15";
 			reg = <13>;
+			clock-frequency= <0>;
 		};
 
 		cpu@14 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a15";
 			reg = <14>;
+			clock-frequency= <0>;
 		};
 
 		cpu@15 {
 			device_type = "cpu";
 			compatible = "arm,cortex-a15";
 			reg = <15>;
+			clock-frequency= <0>;
 		};
 	};
 
-- 
1.8.1.2

